common-lib/src/main/scala/com/gu/mediaservice/model/Image.scala

```diff
@@ -30,6 +30,8 @@ case class Image(
 ) {
   def rcsPublishDate: Option[DateTime] = syndicationRights.flatMap(_.published)
 
+  def isInferred: Boolean = syndicationRights.forall(_.isInferred == true)
```

Comment:
Nitpicking but `isInferred` might be a confusing name on the whole `Image`. Maybe `hasInferredRights`?